src/assets/stylesheets/index.scss

```diff
@@ -216,8 +216,8 @@ fieldset:disabled .btn {
 // Base button variant styles using CSS custom properties
 .btn-primary,
 .btn-secondary {
-  --btn-color-lighten: hsl(from var(--btn-color) h s calc(l + 8));
-  --btn-color-darken: hsl(from var(--btn-color) h s calc(l - 8));
+  --btn-color-lighten: color-mix(in oklch, var(--btn-color), white 18%);
```

Comment:
There are at least four more cases of `hsl` usage in `index.scss`. I would prefer to always use the same syntax.